boards/silabs/radio_boards/siwx917_rb4338a/siwx917_rb4338a.dts

```diff
@@ -82,6 +82,14 @@
 	};
 };
 
+&udma0 {
+	status = "disabled";
+};
+
+&udma1 {
+	status = "disabled";
+};
```

Comment:
Do these really need to be here? The SoC already sets them to disabled, so these shouldn't have any effect.

Reply:
Removed from the dts file